netmanager/src/views/components/MaintenanceBanner/MaintenanceBanner.js

```
@@ -0,0 +1,70 @@
+  return (
+    <Paper elevation={0} className={classes.banner}>
+      <Box className={classes.content}>
+        <WarningIcon className={classes.icon} />
+        <Typography variant="body1" className={classes.message}>
+          {maintenance.message ||
+            'System maintenance in progress. Some features may be unavailable.'}
+        </Typography>
+      </Box>
+      <Typography variant="body2" className={classes.timeInfo}>
+        Estimated downtime:{' '}
+        {maintenance.startDate && !isNaN(new Date(maintenance.startDate))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)